src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double lowerBase;
+    private double upperBase;
+    private double side;
+
+    public IsoscelesTrapezoid(Color color, double lowerBase, double upperBase, double side) {
```

Comment:
Creating an isosceles trapezoid, you should avoid passing expressions directly in the constructor. Instead, create separate variables before the constructor call and pass them to improve readability.